winpr/libwinpr/sspi/NTLM/ntlm.c

```diff
@@ -114,26 +110,21 @@ int ntlm_SetContextServicePrincipalNameA(NTLM_CONTEXT* context, char* ServicePri
 int ntlm_SetContextTargetName(NTLM_CONTEXT* context, char* TargetName)
 {
 	int status;
-	DWORD nSize = 0;
+	TCHAR computerName[MAX_COMPUTERNAME_LENGTH + 1];
```

Comment:
Since you're explicitly using GetComputerNameExA and _strdup below you should not use TCHAR but CHAR here.